### PR TITLE
Fixed entity relationship mapping for persist/remove/orphan

### PR DESCRIPTION
**Pull request by asgrim**
Fixing the bug.

### CODE REVIEW

**Comment by asgrim** [comment]
don't merge yet

File: src/App/Entity/Meetup.php [hunk 1]
```diff
@@ -63,7 +63,12 @@
     private $talks;
 
     /**
-     * @ORM\OneToMany(targetEntity=MeetupAttendee::class, mappedBy="meetup")
+     * @ORM\OneToMany(
+     *     targetEntity=MeetupAttendee::class,
+     *     mappedBy="meetup",
+     *     orphanRemoval=true,
+     *     cascade={"persist", "remove"}
```
Comment (lboynton) [inline src/App/Entity/Meetup.php]: I haven't used orphan removal, so I'm not familiar with it. Does it make sense to use it with cascade remove?

Comment (asgrim) [inline src/App/Entity/Meetup.php]: orphan removal is actually what fixes this issue. Probably don't need the cascade

Comment (lboynton) [inline src/App/Entity/Meetup.php]: :+1:

**Pull request merged**